protocol-units/da-sequencer/node/src/block.rs

```diff
@@ -18,17 +23,71 @@ impl SequencerBlockDigest {
 	}
 }
 
-#[derive(Clone, Copy, Default, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
+#[derive(
+	Serialize, Deserialize, Clone, Copy, Default, Debug, PartialEq, Eq, Hash, PartialOrd, Ord,
+)]
 pub struct BlockHeight(pub u64);
 
-#[derive(Clone, Default, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
+#[derive(Serialize, Deserialize, Clone, Default, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
 pub struct SequencerBlock {
-	height: BlockHeight,
-	block: Block,
+	pub height: BlockHeight,
+	pub block: Block,
```

Comment:
Since `SequencerBlock` has a size restriction enforced by `SequencerBlock::try_new`, should these not be made public and therefore freely modifiable?

Reply:
That was done #1156 and merged in here.

Reply:
Never late to correct a wrong? It's a minor thing though.